virttest/storage.py

```diff
@@ -191,6 +193,18 @@ def get_image_filename(params, root_dir, basename=False):
             address = params['nvme_pci_address']
             namespace = params.get('nvme_namespace', 1)
             return nvme.get_image_filename(address, namespace)
+        if enable_ssh:
+            # required ssh options
+            server = params['ssh_server']
```

Comment:
And also suggest appending the prefix `image_` for these newly introduced params to avoid name conflicts with those ones for command line interfaces.

Reply:
done